tests/Doctrine/Tests/ORM/Tools/Pagination/LimitSubqueryWalkerTest.php

```diff
@@ -47,7 +47,21 @@ public function testLimitSubqueryWithSortFunction() : void
         $limitQuery->setHint(Query::HINT_CUSTOM_TREE_WALKERS, [LimitSubqueryWalker::class]);
 
         self::assertSame(
-            'SELECT DISTINCT m0_.id AS id_0 FROM MyBlogPost m0_ INNER JOIN Category c1_ ON m0_.category_id = c1_.id GROUP BY m0_.id ORDER BY COUNT(c1_.id) ASC',
+            'SELECT DISTINCT m0_.id AS id_0, COUNT(c1_.id) AS sclr_1 FROM MyBlogPost m0_ INNER JOIN Category c1_ ON m0_.category_id = c1_.id GROUP BY m0_.id ORDER BY COUNT(c1_.id) ASC',
```

Comment:
This line is getting long, how about adding some linebreaks?
```suggestion
            <<<SQL
      SELECT DISTINCT m0_.id AS id_0, COUNT(c1_.id) AS sclr_1 
         FROM MyBlogPost m0_ 
INNER JOIN Category c1_ ON m0_.category_id = c1_.id 
 GROUP BY m0_.id 
 ORDER BY COUNT(c1_.id) ASC
 SQL
            ,
```

Reply:
I wanted to keep consistency with others queries.
I we want to rewrite queries on multilines, we should do it everywhere.

Reply:
Not all queries should be multiline, just the long ones.

Reply:
Yes, but if you look at others existing queries in the same file, they are all long queries.
That's why I think it should be done in a separate PR if you want to rewrite all queries on multiline.

Reply:
That's an issue I often encounter as a maintainer: we make a style mistake, and then it's repeated over and over for consistency's sake, until someone goes through the painful task of rewriting it all properly. And the longer you wait, the more painful the task is (for instance here 10 queries to rewrite before, 13 after). And often, there is no actual rule in the style guide that says we should write things in some kind of way. In that case, people probably did not know any better. 

Luckily, someone already broke the unworded rule:
https://github.com/doctrine/orm/blob/21bc805dc9662ff05f3038b30a29e05591838a29/tests/Doctrine/Tests/ORM/Functional/LifecycleCallbackTest.php#L190-L199
So don't worry with being inconsistent with other occurrences, the project is already inconsistent.